kategory-optics/src/test/kotlin/kategory/optics/LensLaws.kt

```diff
@@ -0,0 +1,26 @@
+package kategory.optics
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import kategory.Applicative
+import kategory.Eq
+import kategory.Law
+import kategory.Lens
+import kategory.compose
+import kategory.exists
+import kategory.identity
+
+object LensLaws {
+
+    inline fun <A, B, reified F> laws(lens: Lens<A, B>, aGen: Gen<A>, bGen: Gen<B>, funcGen: Gen<(B) -> B>, EQA: Eq<A>, EQB: Eq<B>, FA: Applicative<F>): List<Law> = listOf(
+            Law("getSet", { forAll(aGen, { a -> EQA.eqv(lens.set(lens.get(a))(a), a) }) }),
```

Comment:
Could these please be separated into functions? They'll read better and you can test them individually.

Reply:
Thanks! I made the changes. Shall I also move it to `kategory-test`?

Reply:
@raulraja What do you think about merging all tests/laws into a single package? Right now it's supposed to be split per module (effects-tests, recursion-tests, optics-tests) but given that it's not a deployable artifact I don't see any problem making it depend on all the modules and contain all laws.